### PR TITLE
i18n: import

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: lighthouse-core/lib/i18n/locales/ar-XB.json [hunk 1]
```diff
@@ -1,117 +1,90 @@
 {
   "lighthouse-core/audits/accessibility/accesskeys.js | description": {
-    "message": "‏‮Access‬‏ ‏‮keys‬‏ ‏‮let‬‏ ‏‮users‬‏ ‏‮quickly‬‏ ‏‮focus‬‏ ‏‮a‬‏ ‏‮part‬‏ ‏‮of‬‏ ‏‮the‬‏ ‏‮page‬‏. ‏‮For‬‏ ‏‮proper‬‏ ‏‮navigation‬‏, ‏‮each‬‏ ‏‮access‬‏ ‏‮key‬‏ ‏‮must‬‏ ‏‮be‬‏ ‏‮unique‬‏. [‏‮Learn‬‏ ‏‮more‬‏](https://web.dev/accesskeys/)."
+    "message": "‏‮Access‬‏ ‏‮keys‬‏ ‏‮let‬‏ ‏‮users‬‏ ‏‮quickly‬‏ ‏‮focus‬‏ ‏‮a‬‏ ‏‮part‬‏ ‏‮of‬‏ ‏‮the‬‏ ‏‮page‬‏. ‏‮For‬‏ ‏‮proper‬‏ ‏‮navigation‬‏, ‏‮each‬‏ ‏‮access‬‏ ‏‮key‬‏ ‏‮must‬‏ ‏‮be‬‏ ‏‮unique‬‏. [‏‮Learn‬‏ ‏‮more‬‏](‏‮https‬‏://‏‮dequeuniversity‬‏.‏‮com‬‏/‏‮rules‬‏/‏‮axe‬‏/3.1/‏‮accesskeys‬‏?‏‮application‬‏=‏‮lighthouse‬‏)."
```
Comment (connorjclark) [inline lighthouse-core/lib/i18n/locales/ar-XB.json:3]: @exterkamp  ??

File: lighthouse-core/lib/i18n/locales/ar-XB.json [hunk 2]
```diff
@@ -1682,205 +1001,64 @@
...
-    "message": "‏‮There‬‏ ‏‮are‬‏ ‏‮a‬‏ ‏‮number‬‏ ‏‮of‬‏ ‏‮WordPress‬‏ ‏‮plugins‬‏ ‏‮that‬‏ ‏‮can‬‏ ‏‮help‬‏ ‏‮you‬‏ [‏‮inline‬‏ ‏‮critical‬‏ ‏‮assets‬‏](https://wordpress.org/plugins/search/critical+css/) ‏‮or‬‏ [‏‮defer‬‏ ‏‮less‬‏ ‏‮important‬‏ ‏‮resources‬‏](https://wordpress.org/plugins/search/defer+css+javascript/). ‏‮Beware‬‏ ‏‮that‬‏ ‏‮optimizations‬‏ ‏‮provided‬‏ ‏‮by‬‏ ‏‮these‬‏ ‏‮plugins‬‏ ‏‮may‬‏ ‏‮break‬‏ ‏‮features‬‏ ‏‮of‬‏ ‏‮your‬‏ ‏‮theme‬‏ ‏‮or‬‏ ‏‮plugins‬‏, ‏‮so‬‏ ‏‮you‬‏ ‏‮will‬‏ ‏‮likely‬‏ ‏‮need‬‏ ‏‮to‬‏ ‏‮make‬‏ ‏‮code‬‏ ‏‮changes‬‏."
+    "message": "‏‮There‬‏ ‏‮are‬‏ ‏‮a‬‏ ‏‮number‬‏ ‏‮of‬‏ ‏‮WordPress‬‏ ‏‮plugins‬‏ ‏‮that‬‏ ‏‮can‬‏ ‏‮help‬‏ ‏‮you‬‏ [‏‮inline‬‏ ‏‮critical‬‏ ‏‮assets‬‏](‏‮https‬‏://‏‮wordpress‬‏.‏‮org‬‏/‏‮plugins‬‏/‏‮search‬‏/‏‮critical‬‏+‏‮css‬‏/) ‏‮or‬‏ [‏‮defer‬‏ ‏‮less‬‏ ‏‮important‬‏ ‏‮resources‬‏](‏‮https‬‏://‏‮wordpress‬‏.‏‮org‬‏/‏‮plugins‬‏/‏‮search‬‏/‏‮defer‬‏+‏‮css‬‏+‏‮javascript‬‏/). ‏‮Beware‬‏ ‏‮that‬‏ ‏‮optimizations‬‏ ‏‮provided‬‏ ‏‮by‬‏ ‏‮these‬‏ ‏‮plugins‬‏ ‏‮may‬‏ ‏‮break‬‏ ‏‮features‬‏ ‏‮of‬‏ ‏‮your‬‏ ‏‮theme‬‏ ‏‮or‬‏ ‏‮plugins‬‏, ‏‮so‬‏ ‏‮you‬‏ ‏‮will‬‏ ‏‮likely‬‏ ‏‮need‬‏ ‏‮to‬‏ ‏‮make‬‏ ‏‮code‬‏ ‏‮changes‬‏."
```
Comment (connorjclark) [inline lighthouse-core/lib/i18n/locales/ar-XB.json:1029]: these urls got so messed up

**Comment by brendankenny** [comment]
also:

>  46 translated locale files
>  305/651 fully translated messages
>  24/651 partially translated messages
>  322/651 untranslated messages

vs current master:

>  46 translated locale files
>  628/651 fully translated messages
>  23/651 untranslated messages

File: lighthouse-core/lib/i18n/locales/ar-XB.json (hunk 1 above)
Comment (connorjclark) [reply lighthouse-core/lib/i18n/locales/ar-XB.json:3]: ok the URL is super old, so maybe it just fell back to the old version of the string, which was  from before us improving the i18n pipeline

**Comment by exterkamp** [comment]
I just did an import and I am getting [wildly different, but seemingly better, results](https://github.com/GoogleChrome/lighthouse/pull/11226). This looks like old content, is your client synced to latest HEAD?

File: lighthouse-core/lib/i18n/locales/en-XA.json [hunk 3]
```diff
@@ -1,117 +1,90 @@
 {
   "lighthouse-core/audits/accessibility/accesskeys.js | description": {
-    "message": "[Åççéšš ķéýš ļéţ ûšéŕš qûîçķļý ƒöçûš å þåŕţ öƒ ţĥé þåĝé. Föŕ þŕöþéŕ ñåvîĝåţîöñ, éåçĥ åççéšš ķéý mûšţ бé ûñîqûé. ᐅ[ᐊĻéåŕñ möŕéᐅ](https://web.dev/accesskeys/)ᐊ. one two three four five six seven eight nine ten eleven twelve thirteen fourteen fifteen sixteen seventeen eighteen nineteen twenty]"
+    "message": "[Åççéšš ķéýš ļéţ ûšéŕš qûîçķļý ƒöçûš å þåŕţ öƒ ţĥé þåĝé. Föŕ þŕöþéŕ ñåvîĝåţîöñ, éåçĥ åççéšš ķéý mûšţ бé ûñîqûé. [Ļéåŕñ möŕé](ĥţţþš://ðéqûéûñîvéŕšîţý.çöm/ŕûļéš/åxé/3.1/åççéššķéýš¿åþþļîçåţîöñ=ļîĝĥţĥöûšé). one two three four five six seven eight nine ten eleven twelve thirteen fourteen fifteen sixteen seventeen eighteen nineteen twenty twentyone twentytwo twentythree twentyfour twentyfive twentysix twentyseven]"
```
Comment (exterkamp) [inline lighthouse-core/lib/i18n/locales/en-XA.json:3]: `ĥţţþš://ðéqûéûñîvéŕšîţý.çöm/ŕûļéš/åxé/3.1/åççéššķéýš¿åþþļîçåţîöñ=ļîĝĥţĥöûšé` This is how you know it's definitely messed up, links should never be accented in en-XA.

**Comment by connorjclark** [comment]
my mistake, I messed up the import. should be good now.

**Pull request merged**